### PR TITLE
Change config on dev-1.18 branch to prepare for v1.18 release

### PR DESCRIPTION
**Pull request by VineethReddy02**
As a part of Kubernetes 1.18 release work.

Signed-off-by: vineeth vineethpothulapati@outlook.com

@jimangel @zacharysarah

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -84,10 +84,10 @@ announcement = false
...
 [[params.versions]]
-fullversion = "v1.17.0"
-version = "v1.17"
-githubbranch = "v1.17.0"
-docsbranch = "release-1.17"
+fullversion = "v1.18.0"
+version = "v1.18"
+githubbranch = "v1.18.0"
+docsbranch = "release-1.18"
 url = "https://kubernetes.io"
```
Comment (sftim) [inline config.toml]: @VineethReddy02 I think you should also add in a new block for v1.17 before v1.16, below

**Comment by sftim** [comment]
/retitle Change config on dev-1.18 branch to prepare for v1.18 release

Hope that's OK.

**Comment by sftim** [comment]
@VineethReddy02 if the /approve doesn't go through, you can self-approve this

**Comment by VineethReddy02** [comment]
@zacharysarah @jimangel Can we merge this?

**Pull request merged**